README.md

```diff
@@ -220,24 +228,6 @@ jenkins:
               workDirPath: "/tmp"
 ```
 
-## Documentation
-
-<a name="handling-secrets"></a>
```

Comment:
Broke links from https://github.com/jenkins-infra/jenkins.io/search?q=handling-secrets that linked to this anchor.